Update doc loop op (#2337)
onnx/onnx: docs/Changelog.md

        const int max_trip_count = 10; // Analogous to input M
        int user_defined_vals[]; // Imagine this is resizable
        /* End implicitly-defined code */
        /* initialize loop-carried variables and scan-output variables */
        bool keepgoing_out = keepgoing
        int b_out = b
  
        for (int i=0; i < max_trip_count && keepgoing_out; ++i) {
          /* Implicitly-defined code: bind actual parameter values
             to formal parameter variables of loop-body */
          bool keepgoing_in = keepgoing_out; 
          bool b_in = b_out;
  
          /* User-defined code (loop body) */
          int my_local = a + b_in; // Reading value "a" from the enclosing scope is fine
          b_out = a - b_in;
          keepgoing_out = my_local > b_out; 
          user_defined_val = b_in + b_in; // b_in and b_out are different variables
          /* End user-defined code */
  
          /* Implicitly defined-code */
          user_defined_vals[i] = user_defined_val // accumulate scan-output values
        }
        // int t = my_local; // Can't do this. my_local is not accessible here.
  

>         // These below values are live-out from the loop and therefore accessible

Reviewer: [nit]
```suggestion
        // These below values are bound to the outputs of the loop and therefore accessible
```
